examples/extending_models/models/models.gen.go

```diff
@@ -240,7 +243,7 @@ func (p *pgClientImpl) listDog(
 			return nil, p.client.errorConverter(&unstable.NotFoundError{
 				Msg: "GetDog: record not found",
 			})
-		} else {
+		} else if !opt.SucceedOnPartialResults || len(ret) == 0 {
```

Comment:
Why do we want the `|| len(ret) == 0`? Isn't that dead code since we already know that `len(ids) > 0` by now (due to the check on line 204) and we know that `len(ids) != len(ret)` because of the check on line 241.

Reply:
didn't notice the line 204 check, dropping this